rpc/server.go

```diff
@@ -419,14 +422,12 @@ func NewServer(logger golog.Logger, opts ...ServerOption) (Server, error) {
 			if err != nil {
 				return nil, err
 			}
-			var loopbackIfc net.Interface
+			var loopbackIfaces []net.Interface
 			for _, ifc := range ifcs {
-				if (ifc.Flags&net.FlagUp) == 0 ||
-					(ifc.Flags&net.FlagLoopback) == 0 ||
-					(ifc.Flags&net.FlagMulticast) == 0 {
+				if (ifc.Flags&net.FlagUp) == 0 || (ifc.Flags&net.FlagLoopback) == 0 {
```

Comment:
So by default linux doesn't have the "multicast" flag set on the loopback interface, so I removed that check. It appears that you can join a group on an interface without that being set explicitly, and all tests still pass.